src/modules/markets/markets.controller.ts

```diff
@@ -17,9 +17,14 @@ export class MarketsController {
   @ApiResponse({
     status: 200,
     description: 'All markets',
-    type: MarketEntity,
+    type: GetMarketsDto,
```

Comment:
This breaks OpenApi spec. Maybe it can keep using MarketEntity, as oecdRiskCategory is number in MarketEntity.

Not sure if we need GetMarketsDto.

Reply:
I think we need a way to differentiate between the result from the stored procedure (which is a string) and the result we want to give from the API (which is a number). Using 2 different classes is probably the easiest way to do it

Either `MarketEntity` or `MarketsService` needs a change though because `marketsService.findAll` is supposed to return `Promise<MarketEntity[]>` but it returns `oecdRiskCategory` as a string when `MarketEntity` says it should be a number

Reply:
@IgnatG sorry, I didn't see your update before my comment 

My concern with using only 1 class for it is that it means our type system is wrong and incomplete:
- the return type of `MarketsService` is incorrect because it returns the wrong type of `oecdRiskCategory`
- we don't have a class / type to describe the result from the stored procedure

Reply:
"the return type of MarketsService is incorrect because it returns the wrong type of oecdRiskCategory"
  * Maybe moving type change to MarketsService would fix issue above
  
"we don't have a class / type to describe the result from the stored procedure" 
  * It is the case for all SPs. We don't have class/type to describe SP result, but TypeOrm entity is used describe both, also to rename all SP field names to API field names. This is not standard approach.

Reply:
Thanks @IgnatG, I'm happy with the change. I'll leave it to @avaitonis to resolve this comment when he is happy too